2_process/src/process_and_style.R

```diff
@@ -1,17 +1,17 @@
-process_data <- function(site_data_filename){
-  nwis_data<- read_csv(site_data_filename)
-  nwis_data_clean <- rename(nwis_data, water_temperature = X_00010_00000) %>% 
+process_data <- function(site_data, fileout){
+  nwis_data_clean_csv <- rename(site_data, water_temperature = X_00010_00000) %>% 
```

Comment:
same idea here as I wrote below: `nwis_data_clean_csv` is a good name for a file _target_. Right now, you are creating an object with the data.frame of actual cleaned data. So you don't need the `_csv` appended to the object name.